fdbserver/DataDistributionTracker.actor.cpp

```diff
@@ -299,11 +299,14 @@ ACTOR Future<Void> readHotDetector(DataDistributionTracker* self) {
 			state Transaction tr(self->cx);
 			loop {
 				try {
-					Standalone<VectorRef<KeyRangeRef>> readHotRanges = wait(tr.getReadHotRanges(keys));
+					Standalone<VectorRef<ReadHotRangeWithMetrics>> readHotRanges = wait(tr.getReadHotRanges(keys));
 					for (auto& keyRange : readHotRanges) {
 						TraceEvent("ReadHotRangeLog")
-						    .detail("KeyRangeBegin", keyRange.begin)
-						    .detail("KeyRangeEnd", keyRange.end);
+						    .detail("ReadDensity", keyRange.density)
```

Comment:
If there are lots of read hot shards, we will see events are throttled. 

A summarized trace events on how many readHotRanges are there can be helpful. Also it may be better to output the hottest key ranges first so that we get the worst hot shard in case events are throttled.

Reply:
A ranking log might be helpful, but my argument is that this read hot log should ideally only report the cases that brings down or is about to bring down an SS process. In other words one should not expect to see many lines of this log since it should only log extreme cases here.

Reply:
I would probably open more PRs to play around with knobs to find a good value to make sure the log works in the way that I described above.